src/cmd/commit.ts

```diff
@@ -39,30 +98,21 @@ const gitPush = async (): Promise<boolean> => {
   }
 }
 
-const getGiteaHealthUrl = async (): Promise<string> => {
-  const d = terminal(`cmd:${cmdName}:getGiteaHealthUrl`)
-  const healthUrl = (await $`git -C ${env.ENV_DIR} config --get remote.origin.url`).stdout.trim()
-  d.debug('gitea healthUrl: ', healthUrl)
-  return healthUrl
-}
-
-const commitAndPush = async (remote): Promise<void> => {
+const commitAndPush = async (): Promise<void> => {
   const d = terminal(`cmd:${cmdName}:commitAndPush`)
   d.info('Committing values')
   const argv = getParsedArgs()
   cd(env.ENV_DIR)
-  await $`git add -A`
   try {
+    await $`pwd && ls -aln && git add -A`
```

Comment:
What is the reason of calling `pwd && ls`

Reply:
??

Reply:
removed